src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/integration/wiremock/PrisonerSearchApiMockServer.kt

```diff
@@ -37,16 +48,7 @@ class PrisonerSearchApiMockServer : WireMockServer(8100) {
              [
                 {
                   "prisonerNumber": "$prisonerNumber",
-                  "pncNumber": "96/371915Q",
-                  "pncNumberCanonicalShort": "96/371915Q",
-                  "pncNumberCanonicalLong": "1996/371915Q",
-                  "croNumber": "177155/96K",
                   "bookingId": "1138058",
-                  "bookNumber": "W73895",
-                  "firstName": "EMANETTA",
-                  "middleNames": "PATES",
-                  "lastName": "ABOLD",
-                  "dateOfBirth": "1984-08-22",
                   "legalStatus": "SENTENCED"
                 }
```

Comment:
None of these details are being used and they wouldn't appear in the real response from prisoner search. I was a little confused why some fields that are being used aren't returned in the stub (e.g. conditionalReleaseDate, recall etc.) but it seems these are being tested in unit tests so don't need them for the integration tests.